### DISCUSSION TITLE
Hope that superset can add the kline type chart

### DISCUSSION THREAD

**Discussion by thewaytoai** [apache/superset#18531] [Ideas]

**Comment by request-info** [comment]
We would appreciate it if you could provide us with more info about this issue/pr! Please do not leave the `title` or `description` empty.

**Comment by thewaytoai** [comment]
I don't know how to integrate the kline with superset, and i couldn't see it within the superset, hope that superset can add the kline type chart

**Comment by JesseLivingston** [comment]
Maybe you can use box chart.
